src/common/utility.js

```diff
@@ -58,6 +58,21 @@ const sanitize = input => input.replace(/[.*+?^${}()|[\]\\]/g, '')
 
 const sentenceCase = input => input.charAt(0).toUpperCase() + input.slice(1)
 
+const snackbar_options = {
+    position: 'bottom-right',
+    style: {
+        backgroundColor: '#ffc71b',
+        color: '#0e0e0e',
```

Comment:
Is it better to use naming colors from `theme.js`?

Reply:
themes color can only injected on the children inside provider